index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
-
-var proto = Element.prototype;
+var Element = require('global-object').Element;
+var proto = Element && Element.prototype || {};
```

Comment:
We can just use `typeof Element !== 'undefined' ? Element.prototype : {}` here.  No need for references to the global object.